lib/svd_quda.h

```diff
@@ -291,6 +291,9 @@ void getRealBidiagMatrix(const Matrix<Cmplx,3> & mat,
   typename RealTypeId<Cmplx>::Type beta;
   Cmplx w, tau, z;
 
+  u(0,0).x = -2.0;
```

Comment:
Ok, I've cleaned up this up and added clarification in the comments what's going on here.